src/test/scala/com/mozilla/telemetry/TestUtils.scala

```diff
@@ -205,7 +205,7 @@ object TestUtils {
       "payload.simpleMeasurements" ->
         """
           |{
-          |  "activeTicks": 2000,
+          |  "activeTicks": 275,
```

Comment:
Putting in a value slightly above 271 here to mimic the activeTicks overcounting bug. This number should never show up in the tests, because the scalar version should get picked up instead.